src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -0,0 +1,49 @@
+package nextstep.subway.ui;
+
+import nextstep.subway.application.LineService;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> createStation(@RequestBody LineRequest request) {
+        LineResponse response = lineService.createLine(request);
+        return ResponseEntity.created(URI.create("/lines/" + response.getId())).body(response);
+    }
+
+    @GetMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> getLines() {
+        return ResponseEntity.ok().body(lineService.getLines());
```

Comment:
```suggestion
        return ResponseEntity.ok(lineService.getLines());
```

소소하지만 다음과 같이 사용할 수 있습니다 ㅎㅎ